src/converse-chatview.js

```diff
@@ -91,6 +91,7 @@
                 events: {
                     'click .close-chatbox-button': 'close',
                     'keypress .chat-textarea': 'keyPressed',
+                    'click .send-button': 'clickSendButton',
```

Comment:
`clickSendButton` makes it sound like a click action will be performed, when in reality what's happening is that the method is a response to a click action which has already happened.

So rather name this someting like `onSendButtonClicked`.